frontend/src/components/EmployeeCards/EmployeeCards.tsx

```diff
@@ -74,21 +75,21 @@ const EmployeeCard = ({
    * @param availability the driver's availability, represented as an object map of days to start and end times
```

Comment:
You can delete this since we arent displaying it on the card anymore no need to comment it out